tests/store/tracking/test_file_store.py

```diff
@@ -402,10 +408,14 @@ def test_create_experiment(self):
             exp1.artifact_location,
             path_to_local_file_uri(posixpath.join(self.test_root, created_id)),
         )
+        assert isinstance(exp1.creation_time, int)
+        assert isinstance(exp1.last_update_time, int)
```

Comment:
```suggestion
        assert exp1.creation_time >= time_before_create 
        assert exp1.creation_time == exp1.last_update_time
```

Can we make this more strict?

Reply:
done